tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -156,6 +164,29 @@ trait ExpectHelpers extends FunSuiteLike {
     // buf.toString
     owners.toList.sortBy(_._1).map(kv => s"${kv._1} => ${kv._2}").mkString(EOL)
   }
+
+  protected def indexSyntax(path: Path): String = {
+    val semanticdbSemanticidx = path.resolve("META-INF/semanticdb.semanticidx")
+    if (Files.exists(semanticdbSemanticidx)) {
+      val index = FileIO.readIndex(AbsolutePath(semanticdbSemanticidx))
+      val buf = new StringBuilder
+      buf.append("Packages:" + EOL)
+      buf.append("=========" + EOL)
+      index.packages.sortBy(_.symbol).foreach { entry =>
+        buf.append(entry.symbol + EOL)
+        entry.members.sorted.foreach(member => buf.append("  " + member + EOL))
+      }
+      buf.append(EOL)
+      buf.append("Toplevels:" + EOL)
+      buf.append("==========" + EOL)
+      index.toplevels.sortBy(_.symbol).foreach { entry =>
+        buf.append(s"${entry.symbol} => ${entry.uri}" + EOL)
```

Comment:
Seems we need `PathIO.toUnix(entry.uri)` on the right hand side to please Appveyor.